SONIC-DEV/sonic-buildimage
# [device/alibaba]: Add BMC/BIOS upgrade logger and fix bug

**- What I did**
- Add BMC/BIOS upgrade logger and fix conflict logger bug

**- How I did it**
- Remove old  BMC/BIOS upgrade logger
- Add BMC/BIOS upgrade logger as same as CPLD/FPGA upgrade logger

**- How to verify it**
- Upgrade BMC/BIOS via `fwmgrutil`
- Check upgrade log in `/var/log/fw_upgrade.log`
